Cargo.lock

```diff
@@ -5760,9 +5760,9 @@ dependencies = [
 
 [[package]]
 name = "serde_json"
-version = "1.0.138"
+version = "1.0.139"
```

Comment:
Bumped due to `solana-sdk` update and its dependency requirements: https://crates.io/crates/solana-sdk/2.2.2/dependencies